### PR TITLE
Fix GetWinSize for Windows

### PR DESCRIPTION
**Pull request by sttts**
Fixes #25273 on Windows.

### CODE REVIEW

**Comment by sttts** [comment]
/cc @ncdc 

I tested the code with a small 10 liner on a Windows VM. I have **not** tested the actual kuberctl with this patch.

File: pkg/util/term/term.go [hunk 1]
```diff
@@ -119,18 +119,13 @@ type Size struct {
 // GetSize returns the current size of the user's terminal. If t.In is nil or it doesn't have a file
```
Comment (sttts) [inline pkg/util/term/term.go:119]: The comment is wrong now.

**Comment by sttts** [comment]
/cc @csrwng 

Can you test this branch on Windows?

**Comment by csrwng** [comment]
@sttts I tested this on Friday and got an error message when starting the exec, but I realized I wasn't using the same branch on the server side. I'm hoping I can set it up today so client and server match. Will let you know how it goes.

**Comment by csrwng** [comment]
@sttts I'm getting this:

```
C:\Users\cewong\go\src\k8s.io\kubernetes>.\kubectl.exe exec -ti nginx-3137573019-nocjn -- /bin/bash
E0607 10:27:54.638312    2192 term.go:135] unable to get terminal size: The handle is invalid.
root@nginx-3137573019-nocjn:/#
```

**Pull request closed without merging**